aini.go

```diff
@@ -78,16 +78,37 @@ func Parse(r io.Reader) (*InventoryData, error) {
 }
 
 // Match looks for a hosts that match the pattern
-func (inventory *InventoryData) Match(m string) []*Host {
+func (inventory *InventoryData) Match(pattern string) []*Host {
 	matchedHosts := make([]*Host, 0)
 	for _, host := range inventory.Hosts {
-		if m, err := path.Match(m, host.Name); err == nil && m {
+		if m, err := path.Match(pattern, host.Name); err == nil && m {
```

Comment:
Should `err` be returned here, since it indicates `pattern` is invalid? (or log or panic...?)